src/OSPSuite.SimModelNative/CMakeLists.txt

```diff
@@ -5,6 +5,13 @@ set (CMAKE_CXX_STANDARD 17)
 set (CMAKE_CXX_FLAGS_DEBUG "-g")
 set (CMAKE_CXX_FLAGS_RELEASE "-O3")
 
+if (APPLE)
+    set(CMAKE_MACOSX_RPATH 1)
+    set (CMAKE_CXX_FLAGS_RELEASE "-O0")
```

Comment:
This is the flag that needed to change to allow tests to pass.  This reduces the optimization level for C++ code on Mac builds.

There must be some differences in the clang vs gcc implementation.

Reply:
A change in a txt file.   Crazy

Reply:
It could also be a difference in the clang version. On my Mac M3 the clang version is 16, on AppVeyor it's 14.

Reply:
ok, turns out I don't need that flag if I update the AppVeyor image to something newer. I didn't know there was an option.